model-server-test/build.gradle.kts

```diff
@@ -0,0 +1,40 @@
+import org.gradle.nativeplatform.platform.internal.DefaultNativePlatform
+
+plugins {
+    kotlin("jvm")
+    alias(libs.plugins.docker.compose)
+}
+
+dependencies {
+    testImplementation(kotlin("test"))
+    testImplementation(project(":model-server"))
+}
+
+tasks.test {
+    doFirst {
+        val db = dockerCompose.servicesInfos.getValue("db")
+        systemProperty("jdbc.url", "jdbc:postgresql://${db.host}:${db.port}/")
```

Comment:
We would still have to use system properties to pass the correct port number from Gradle to the test code.

Reply:
If `IgniteStoreClient` did nor rely on our properties names, we could use the properties or environment variables that the plugin sets directly in the test when creating an `IgniteStoreClient`.

(Here it does not make a difference. It might be more interesting in tests, where we do not want to have system properties at all.)
 
> * The name of environment variable is ${serviceName}_HOST and ${serviceName}_TCP_${exposedPort} (e.g. WEB_HOST and WEB_TCP_80).
> * The name of Java system property is ${serviceName}.host and ${serviceName}.tcp.${exposedPort} (e.g. web.host and web.tcp.80).
> * If the service is scaled then the serviceName has _1, _2... suffix (e.g. WEB_1_HOST and WEB_1_TCP_80, web_1.host and web_1.tcp.80)

https://github.com/avast/gradle-docker-compose-plugin?tab=readme-ov-file